backend/server/lib/apiregistry_extensions/jwttokenmanager.js

```diff
@@ -28,7 +28,7 @@ function initSync() {
 
     if (TOKENMANCONF.useGlobalMemory) TOKEN_MEMORY = _ => DISTRIBUTED_MEMORY; else TOKEN_MEMORY = _ => CLUSTER_MEMORY;
 
-    try {conf = require(TOKENMANCONF);} catch (err) {conf = {}}
+    try {conf = TOKENMANCONF;} catch (err) {conf = {}}
```

Comment:
Catch is redundant, secondly having 2 variables - conf and TOKENMANCONF is also wrong.

If you are submitting a fix - please fix correctly.

Reply:
Hi Rohit,

It is better to remove the conf variable and directly use TOKENMANCONF and we can remove the try-catch block as the conf variable is not required. We can use the below code:

```
TOKENMANCONF.expiryInterval = TOKENMANCONF?.expiryInterval || DEFAULT_TOKEN_EXPIRY; 
    TOKENMANCONF.tokenGCInterval = TOKENMANCONF?.tokenGCInterval || DEFAULT_GC_INTERVAL;

    setInterval(_cleanTokens, TOKENMANCONF.tokenGCInterval);   
    ```
    
    Could you please review the above code?

Reply:
Yes that was my point!

Reply:
Ok, Rohit. Closing this PR and raising a new one.